docs/access-management/rbac.md

```diff
@@ -0,0 +1,54 @@
+---
+products: oss-enterprise, cloud-teams
+---
+
+# Role Based Access Control (RBAC)
+
+Role Based Access Control allows a user with Administrative access to apply roles to users, granting different levels of permission within an organization or workspace. 
+
+## Workspace Resource Roles
+A user can have at most one role of this kind per workspace. Permissions are scoped to the specific workspace in which the user has this role.
+
+| Permissions           | WorkspaceReader    | WorkspaceEditor    | WorkspaceAdmin    |
+| ---------------------- | :--------: | :--------:| :--------: |
+| **ReadWorkspace**<br />-List the connections in a workspace<br />-Read individual connections<br />-Read workspace settings (data residency, users, connector versions, notification settings) | X | X | X |
+| **Update Connection**<br />-Start/cancel syncs<br />- Modify a connection, including name, replication settings, normalization, DBT<br />- Delete a connection<br />- Create/Update/Delete connector builder connectors |  | X | X |
+| **UpdateWorkspace**<br />- Update workspace settings (data residency, users, connector versions, notification settings)<br />- Modify workspace connector versions | |  | X |
+
+## Organization Resource Roles
+
+A user can have at most one role of this kind per organization. Permissions are scoped to the given organization for which the user has this role, and any workspaces within.
+
+| Permissions           | OrganizationMember | OrganizationReader | OrganizationEditor |OrganizationAdmin |
+| :---------------------- | :--------: | :--------: | :--------: |:--------: |
+| **ReadOrganization**<br />- Read individual organizations | X | X | X | X |
+| **CreateWorkspace**<br />- Create new workspace within a specified organization<br />- Delete a workspace | | | X | X |
+| **UpdateOrganization**<br />- Modify organization settings, including billing, PbA, SSO<br />- Modify user roles within the organization | |  |  | X |
+
+## Instance Resource Roles
+
+At the instance level, a user may have InstanceAdmin role. Permissions are valid for all workspaces and all organizations. This user, therefore would have the following permissions as `InstanceAdmin`: 
+
+**ReadWorkspace**
+- List the connections in a workspace
+- Read individual connections
+- Read workspace settings (data residency, users, connector versions, notification settings)
+**Update Connection**
+- Start/cancel syncs
+- Modify a connection, including name, replication settings, normalization, DBT
+- Delete a connection 
+
+**UpdateWorkspace**
+- Update workspace settings (data residency, users, connector versions, notification settings)
+- Modify workspace connector versions
+
+**ReadOrganization**
+- Read individual organizations 
+
+**CreateWorkspace**
+- Create new workspace within a specified organization
+- Delete a workspace 
+
+**UpdateOrganization**
+- Modify organization settings, including billing, PbA, SSO
+- Modify user roles within the organization
```

Comment:
This role cannot be assigned via the UI and is only available for internal users (ie: Airbyters) on Cloud.

In Enterprise, I believe the initial user is granted this permission and the only way that could be removed/changed/etc. is via direct database manipulation.  (@timroes may know for sure who gets that role in Enterprise probably)

Reply:
Thanks! I wasn't sure who to ask about this role. That helps!